doc/design.txt

```diff
@@ -272,7 +272,7 @@ purposes; they are not actually present in the boot vector.
     -----------------+--------+--------'
      swap: none                        |
     -----------------------------------'
-    
```

Comment:
> 3 != 4 (and there are four states listed in the BOOT STATES section)

I agree with this and I think the sentence should be ""It is better to map all the possible vector states to the *four* states described...".